packages/vc-test-suite/helpers.ts

```diff
@@ -35,3 +35,27 @@ export function setupMatrix(implementors: string[], columnLabel: string) {
   const images = [];
   this.images = images;
 }
+
+/**
+ * Builds a URL with additional query parameters.
+ * @param baseUrl - The base URL.
+ * @param additionalParams - Additional query parameters as key-value pairs.
+ * @returns The built URL as a string.
+ */
+export const buildUrl = (baseUrl: string, additionalParams: Record<string, any>) => {
+  const url = new URL(baseUrl);
+  Object.entries(additionalParams).forEach(([key, value]) => {
+    url.searchParams.append(key, value);
```

Comment:
Hi. Should we add String(value) in ` url.searchParams.append(key, String(value));` for ensure values are strings for URL encoding?